metadata.json

```diff
@@ -30,10 +30,10 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 4.9.0 < 7.0.0"
+      "version_requirement": ">= 5.5.10 < 7.0.0"

```

Comment:
Be sure to mark this PR as backwards incompatible since we're doing a major version bump here.